pumps/resurface.go

```diff
@@ -0,0 +1,217 @@
+package pumps
+
+import (
+	"context"
+	"encoding/base64"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/TykTechnologies/tyk-pump/analytics"
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
```

Comment:
@monrax can you remove this dependency? Ideally you'd want to use Go errors pkg.

Reply:
Done